### ISSUE TITLE
Wrap up as electron app?

### ISSUE THREAD

**Issue by sp00x** [ryanwmckenna/vizrt-command-sender#1]
Just a thought, but maybe you could wrap this up as an Electron app (http://electron.atom.io/) and provide some default builds to lower the entrance level threshold for testing it out?

**Comment by ryanwmckenna** [comment]
Yes, agreed. We've just been waiting on getting some time to develop an easier way to execute the application. Should be some updates over the next few weeks.

**Comment by andyleggett** [comment]
Added a default build - available via the readme.  Electron build once I've learned that!
